rustls-platform-verifier/src/verification/others.rs

```diff
@@ -78,18 +78,10 @@ impl Verifier {
             }
         }
 
-        let mut num_extra_roots = 0;
-        let (added, ignored) =
-            root_store.add_parsable_certificates(roots.into_iter().inspect(|_| {
-                num_extra_roots += 1;
-            }));
-
-        if num_extra_roots > 0 {
-            if ignored > 0 {
-                log::warn!("{ignored} extra CA root certificates were ignored due to errors");
-            } else {
-                log::debug!("Loaded {added} CA root certificates from extra roots");
-            }
+        // While we ignore invalid certificates from the system, we forward errors from
```

Comment:
@paolobarbolini added this change. Should help?

Reply:
Perfect. Thanks for the quick fix!